app/decorators/project_decorator.rb

```diff
@@ -26,7 +26,7 @@ def sorted_link_list
   end
 
   # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
-  def sidebar
+  def sidebar(current_user)
```

Comment:
Rename `current_user` to `account` because the actual object is account. Variable naming should reflect intention. Here the intention is passing an `account` object, it doesn't matter if the account object is current_user or some other user.